src/Compositor.hpp

```diff
@@ -85,11 +85,15 @@ class CCompositor {
     wlr_surface*            m_pLastFocus = nullptr;
     CWindow*                m_pLastWindow = nullptr;
     SMonitor*               m_pLastMonitor = nullptr;
+    int                     m_iLastWorkspaceID = -1; // Start with no last workspace, represented by -1.
```

Comment:
why?? is it not better to make a chain??

Reply:
I suppose so, I guess it's just never been relevant to my workflow. I think I must have assumed the `auto_back_and_forth` behaviour as the default.

In my comment below I suggest having each workspace remember the workspace it was accessed from, to form such a chain. Does that sound workable?